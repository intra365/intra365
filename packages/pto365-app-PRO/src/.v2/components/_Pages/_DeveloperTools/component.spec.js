import React, { Component } from 'react';

import _DeveloperToolsPage from '.'

test('Check that it renders', () => {

    const wrapper = shallow(<_DeveloperToolsPage context={mocks.context()}/>);

    expect(wrapper).toMatchSnapshot();
 
}

)