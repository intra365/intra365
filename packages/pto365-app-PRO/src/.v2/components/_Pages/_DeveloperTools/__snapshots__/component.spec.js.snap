// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check that it renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <_DeveloperToolsPage
    context={Object {}}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <PageHeader
          color="red"
          title="*** INTERNAL DEVELOPER TOOLS ***"
/>,
        <PageBody>
          <CustomizedDefaultButton
                    onClick={undefined}
                    text="UP"
          />
          {}
</PageBody>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "color": "red",
          "title": "*** INTERNAL DEVELOPER TOOLS ***",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <CustomizedDefaultButton
              onClick={undefined}
              text="UP"
/>,
            "{}",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "onClick": undefined,
              "text": "UP",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "{}",
        ],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <PageHeader
            color="red"
            title="*** INTERNAL DEVELOPER TOOLS ***"
/>,
          <PageBody>
            <CustomizedDefaultButton
                        onClick={undefined}
                        text="UP"
            />
            {}
</PageBody>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "color": "red",
            "title": "*** INTERNAL DEVELOPER TOOLS ***",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <CustomizedDefaultButton
                onClick={undefined}
                text="UP"
/>,
              "{}",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "onClick": undefined,
                "text": "UP",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "{}",
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
